docs/docs/rest-api/public/api/v2/schema/AppDefinition.json

```diff
@@ -1,712 +0,0 @@
-{
```

Comment:
The json schema should be superseded by the RAML doc. We currently make the json schema available: https://github.com/mesosphere/marathon/blob/master/src/main/scala/mesosphere/marathon/api/v2/SchemaResource.scala#L22

Perhaps we can create the schema from raml? (e.g.https://www.npmjs.com/package/raml-jsonschema-expander)